src/app/pages/kittens/kittens.component.html

```diff
@@ -0,0 +1,40 @@
+<main class="main-grid-container">
+
+    <!-- AVAILABLE kittens section -->
+    <section class="grid-itemList">
+        <h3>Chatons disponibles</h3>
+
+        <!--<button (click)="pushKittenForTest()"> ajouter</button>-->
+        <div class="grid-itemList-CardsContainer">
+
+            @for (kitten of getServiceKittenAvailableKittens(); track $index) {
+            <div class="grid-itemList-CardContainer">
+                <div>
+                    <a (click)="addAdoptedKittenToUserKittenLis($index)">
+                        <!-- Card @Input -->
+                        <app-kitten [kittenCard]="kitten"></app-kitten>
```

Comment:
and organize H5 tag inside this component to be a logical suit to the previous h in this page